docs/user-guide/install-customization.md

```diff
@@ -30,6 +30,42 @@ file=openshift/99-openshift-machineconfig-master-kargs.yaml
 curl --header "Authorization: Bearer $TOKEN" "http://$ASSISTED_SERVICE_IP:$ASSISTED_SERVICE_PORT/api/assisted-install/v2/clusters/$CLUSTER_ID/manifests/files?file_name=$file"
 ```
 
+### Manifests use cases
+
+#### Configure storage on nodes using MachineConfig manifests
+
+Changes to operating systems on OCP nodes can be done by creating what are referred to as MachineConfig objects that are managed by the Machine Config Operator (MCO).
+
+The [MachineConfig specification](https://github.com/openshift/machine-config-operator/blob/master/docs/MachineConfiguration.md) includes an [ignition config](https://github.com/coreos/ignition/blob/main/docs/configuration-v3_2.md) for configuring the machines at first boot.
+This config object can be used to do things like modify files, systemd services, and other operating system features running on on OCP machines.
+
+Specifically, the ignition config can be used to configure storage on the nodes (see ignition [Config Spec](https://coreos.github.io/ignition/configuration-v3_2/) for full details).
+The following MachineSet manifest example demonstrates how to add a partition to `sda` device on master nodes. I.e. apply the manifest before installation to have a partition named `recovery` of size 16 GiB on the master nodes.
+
+```yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 10_masters_storage_config
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      disks:
+        - device: /dev/sda
+          partitions:
+            - label: recovery
+              startMiB: 32768
```

Comment:
Can we explain what startMIB is for, is this an offset on the disk where the partition should be placed?
How does the user come to this calculation, some advice here might be good?

Reply:
A bit of Googling brought me here
https://coreos.github.io/ignition/configuration-v3_4_experimental/

`startMiB (integer): the start of the partition (in mebibytes). If zero, the partition will be positioned at the start of the largest block available.`

It may be an idea to link some documentation that describes documentation on this configuration so that it's clear what options are available?

Reply:
Right, OCP 4.11 supports [Ignition Configuration Specification v3.2.0](https://coreos.github.io/ignition/configuration-v3_2/), see [OCP doc](https://docs.openshift.com/container-platform/4.11/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks). So I've already added in this change a link to that documentation: https://github.com/openshift/assisted-service/pull/4873/files#diff-2ba87ee0005a7fb9cb20a9a3557045068d8e51e358692ed2cb52cb2feca321e0R42